bin/martinize2

```diff
@@ -384,9 +384,9 @@ def entry():
         raise ValueError('Unknown force field "{}".'.format(args.to_ff))
     if args.from_ff not in known_force_fields:
         raise ValueError('Unknown force field "{}".'.format(args.from_ff))
-    if from_ff not in known_mappings or args.to_ff not in known_mappings[from_ff]:
-        raise ValueError('No mapping known to go from "{}" to "{}".'
-                         .format(from_ff, args.to_ff))
+    #if from_ff not in known_mappings or args.to_ff not in known_mappings[from_ff]:
```

Comment:
They're not :)
Instead, you get a list of atoms that are not mapped. It could be that you do define a mapping for e.g. the backbone of a protein, but not its sidechains. That would result in a list of sidechain atoms being printed (as warning).

Reply:
I was thinking about combination of force fields. Is it identified if I ask to go from force field 'XYZ' to force field 'ABC' but 'ABC' does not exist?

Reply:
Probably not. It would just give a huge list of unmapped atoms.